tests/manifests/test-vm.yml

```diff
@@ -20,7 +20,7 @@ objects:
         - name: mydisk
           volumeName: pvcvolume
           disk:
-            dev: vda
+            bus: virtio
```

Comment:
spoke to @cynepco3hahue , dev under the disk does not exist  for a long time now and single reason why it worked before, because we did not have correct validation for unknown fields.
So, updating the above to `bus: virtio`.